tests/python/unit/test_ontology_tagger.py

```diff
@@ -56,8 +59,8 @@ def test_missing_keyword_in_ontology():
     assert isinstance(tagger.matcher_dict[tagger.language], PhraseMatcher)
```

Comment:
Not sure that we need to assert for class instances. 

It's better to assert one thing only for each test, by checking the equality with expected end results

Reply:
1e16f3e and  74a8d5b :)